app/src/main/res/values/strings.xml

```diff
@@ -415,8 +415,14 @@
     <string name="lock_post">Lock Post</string>
     <string name="unlock_post">Unlock Post</string>
     <string name="type_your_reason">Type your reason</string>
+    <string name="disabled_swipe_action_preset">Disabled</string>
```

Comment:
If you want to prefix these, then add that first, IE `swipe_action_preset_disabled`